distros/kubernetes/nvsentinel/charts/mongodb-store/templates/jobs.yaml

```diff
@@ -54,6 +56,21 @@ spec:
         {{- toYaml . | nindent 8 }}
       {{- end }}
       initContainers:
+{{- if index .Values "mongodb-store" "usePerconaOperator" }}
```

Comment:
Can we instead update the existing script to accept the secret name via an env var and only switch the env var based on flags?